CMakeLists.txt.in

```diff
@@ -275,13 +275,17 @@ ExternalProject_Add(lksctp-tools
   INSTALL_COMMAND ${make_command} install)
 
 ExternalProject_Add(seastar
-  URL https://github.com/vectorizedio/seastar/archive/319d359.tar.gz
-  URL_MD5 341f15559319f93463961db03978d45c
+  GIT_REPOSITORY https://github.com/vectorizedio/seastar.git
+  GIT_TAG d80beb8f0e5f926b79db9538e62c32c5d3bcc076
```

Comment:
fantastic, i wonder if we copy all the other libs that `vectorizedio/redpanda` are using too.

Reply:
For sure ! I will handle that in a separate issue.